libs/libc/misc/Kconfig

```
@@ -81,9 +81,27 @@ config LIBC_FTOK_VFS_PATH
 	---help---
 		The relative path to where ftok will exist in the root namespace.
 
+choice
+	prompt "Select memfd implementation"
```

```suggestion
	prompt "Select memfd implementation"
	default LIBC_MEMFD_ERROR
```
? Or what is the default? `LIBC_MEMFD_SHMFS` if `FS_SHMFS`?

---

I don't add the default value intentionally, kconfig tool select the first available choice in this case:

1. If user enable FS_SHMFS, memfd implement on top of shmfs
2. if user disable FS_SHMFS, but enable FS_TMPFS memfd impleemnt on top of tmpfs
3. if user disable both, memfd select LIBC_MEMFD_ERROR

This behavior is most user expect.

---

Ok